terraform/main.tf

```diff
@@ -1,15 +1,6 @@
 ﻿// declarations for the resources to be created 
 // this does not include the state S3 bucket as this must be created in order for terraform to init
 
-
-module "example" {
-  source = "terraform-aws-modules/s3-bucket/aws"
-  
-  bucket = "gpg-bucket-${terraform.workspace}"
-  acl    = "private"
-
-  versioning = {
-    enabled = true
-  }
-
+data "aws_availability_zones" "available" {
+  state = "available"
 }
```

Comment:
Showing how little I know about terraform here, but do we need to do more than this in the `main.tf` file? Or does it just detect there are more `.tf` files and run them all?

Reply:
It detects all .tf files in the root terraform folder so we should be fine.